Source/JavaScriptCore/b3/B3ValueRep.h

```diff
@@ -128,6 +137,12 @@ class ValueRep {
     {
         switch (location.kind()) {
         case Wasm::ValueLocation::Kind::GPRRegister:
+#if USE(JSVALUE32_64)
+            m_kind = RegisterPair;
+            u.registerPair.regHi = location.jsr().tagGPR();
+            u.registerPair.regLo = location.jsr().payloadGPR();
+            break;
+#endif
```

Comment:
The following code looks unreachable on JSVALUE32_64.  Should this be an #if #else case?